test/api/run

```diff
@@ -135,6 +144,12 @@ else
     exit 1
 fi
 
+basedir="$(dirname $0)"
+for test in "$basedir"/*/run.sh; do
```

Comment:
I'm curious why you wanted to call those files basedir/*/run.sh instead of just basedir/*.sh or basedir/*.somethingelse. What do you expect those subdirectories will have besides the run.sh?

Reply:
More importantly, I would seriously consider using pytest as test/{alternator,cql-pytest,redis} do, it's so much more convenient than these test scripts, and you can see the different failures on Jenkins, and so on.

Reply:
np, will use pytest.
For basic tests I expect only run.sh, but we may want to add other files there if needed.